packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -1195,7 +1192,10 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 					id,
 					this.parentContext.IFluidHandleContext,
 				);
-				this.parentContext.addedGCOutboundReference?.(this.containerRuntimeHandle, handle);
+				this.parentContext.addedGCOutboundRoute(
```

Comment:
Is this for super old "root" concept, that's replaced by aliasing?  Why do we still need this code, like what's the scenario?

Reply:
I am removing this in a follow up PR.